collect_app/build.gradle

```diff
@@ -199,6 +199,8 @@ configurations.all {
 }
 
 dependencies {
+    def fragment_version = "1.2.4"
```

Comment:
Can we fallow the same approach like in case of `daggerVersion` and `okhttp3Version` and keep the version in https://github.com/getodk/collect/blob/master/build.gradle#L64?

Reply:
@grzesiek2010 Yes good point! I'm not at my computer right now so happy for you to make and push that change.

Reply:
Ok I'll update it in a minute.